std/machines/large_field/rotate.asm

```diff
@@ -0,0 +1,66 @@
+use std::utils::unchanged_until;
+use std::utils::cross_product;
+use std::convert::int;
+
+// Rotate for single bytes using an exhaustive table
+// TODO this way, we cannot prove anything that rotates by more than 31 bits.
+machine ByteRotate with
+    latch: latch,
+    operation_id: operation_id,
+    degree: 65536
+{
+    operation run<0> P_operation, P_A, P_B, P_ROW -> P_C;
+
+    col fixed latch = [1]*;
+    col fixed operation_id = [0]*;
+
```

Comment:
Can you also add a minimum field size assertion?

Reply:
Inside the Rotate right? Not, inside the ByteRotate